examples/demo-rollup/Cargo.toml

```diff
@@ -67,6 +67,7 @@ ethers-signers = { workspace = true }
 ethers = { workspace = true }
 revm = { workspace = true }
 
+sov-demo-rollup = { path = ".", features = ["experimental"] }
```

Comment:
was this removed only because of issue in evm?

Reply:
Yep, that's correct